app/views/content/help-and-support.md

```diff
@@ -8,6 +8,7 @@ description: |-
 navigation: 40
 layout: "layouts/minimal"
 talk_to_us: false
+breadcrumbs: false
```

Comment:
I don't think this is needed - I know it doesn't have a breadcrumb in production but I think it should unless you disagree @jenhadfield-dfe ?

Reply:
yes agree, would work better with a breadcrumb on this page